manifests/clusterapi-controller.yaml

```diff
@@ -45,14 +45,14 @@ spec:
             cpu: 100m
             memory: 30Mi
       - name: aws-machine-controller
-        image: quay.io/alberto_lamela/aws-machine-controller:mvp # TODO: move this to openshift org
+        image: quay.io/bison/aws-machine-controller:cfc14e5 # TODO: move this to openshift org
```

Comment:
We should be able to use image built by CI now? cc @ingvagabund

Reply:
Can't find the image yet.

Reply:
I see `machine-api-operator` images in the output of `oc describe is -n openshift origin-v3.11 --server https://api.ci.openshift.org` but they reference the internal registry on that cluster. The images are supposed to be mirrored to Quay and Docker Hub, but I'm not sure where to find them.